helix-core/src/test/java/org/apache/helix/integration/rebalancer/TestAbnormalStatesResolver.java

```diff
@@ -64,4 +80,101 @@ public void testConfigureResolver() {
     clusterConfig.setAbnormalStateResolverMap(Collections.emptyMap());
     configAccessor.setClusterConfig(CLUSTER_NAME, clusterConfig);
   }
+
+  @Test(dependsOnMethods = "testConfigureResolver")
+  public void testExcessiveTopStateResolver() {
+    BestPossibleExternalViewVerifier verifier =
+        new BestPossibleExternalViewVerifier.Builder(CLUSTER_NAME).setZkClient(_gZkClient).build();
+    Assert.assertTrue(verifier.verify(5000));
```

Comment:
Use a constant instead of 5000? TestHelper has a constant value built-in.

Reply:
Sure, good point.